tools/gulptasks/dashboards/dist-examples.js

```diff
@@ -69,11 +69,19 @@ async function transformExampleDir(examplesDir) {
 
         if (fileName === 'demo.html') {
             filePath = filePath.replace('demo.html', 'index.html');
-            newContent = [
-                '<link rel="stylesheet" type="text/css" href="./demo.css">',
-                newContent,
-                '<script src="./demo.js"></script>'
-            ].join('\n');
+            newContent = `
```

Comment:
The indent in the result is off. It would be better to continue with array.join.